download_files.py

```diff
@@ -21,7 +21,7 @@
         "CONDA_PKGS_DIRS": os.path.join('repos', 'main', subdir),
 }):
     try:
-        python_api.run_command("create", "-n", "fakeenv", "--download-only", "python", "mkl")
+        python_api.run_command("create", "-n", "fakeenv", "--download-only", "python=3.6", "mkl")
```

Comment:
gives an idea of the age of the solver tests, they probably aren't worth the trouble.